src/utilities/example-group/example-group.factory.ts

```diff
@@ -9,56 +9,62 @@ import OASOperation from '../oas-operation';
 import ExampleGroup from './example-group';
 
 class ExampleGroupFactory {
-  static buildFromOperation(operation: OASOperation): ExampleGroup[] {
+  static buildFromOperation(operation: OASOperation): ExampleGroup[] | [] {
     const parameters = operation.parameters;
-    const groupNames = this.getGroupNames(parameters);
+    if (parameters) {
```

Comment:
Rather than this long `if` block, I think it would be easier to read if we bailed out earlier if there were no parameters...
```ts
    if (!parameters) {
      return [];
    }
```